src/components/MainForm/MainForm.css

```diff
@@ -1,7 +1,3 @@
-body{
-  /* background-color: #e5e5e5; */
-  
-}
 select {
```

Comment:
@IshfaqAhmedAwan why didn't you make it clear that for material ui we don't do styling like this?